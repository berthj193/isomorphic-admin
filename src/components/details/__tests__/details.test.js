import React from 'react';
import Details from '../details';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<PageDetails />', () => {
  const getComponent = (props = {}) => shallow(<Details {...props} />);

  it('should render properly', () => {
    const details = [
      { Hello: 'World' },
      { 'Hello-world': <div>Hello World</div> },
      '---',
    ];
    expect(toJson(getComponent({ details }))).toMatchSnapshot();
  });
});
