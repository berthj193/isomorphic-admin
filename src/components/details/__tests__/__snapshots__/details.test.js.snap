// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageDetails /> should render properly 1`] = `
<Form
  className="details-page"
  hideRequiredMark={false}
  layout="horizontal"
  onSubmit={[Function]}
  prefixCls="ant-form"
>
  <FormItem
    key="0"
    label="Hello"
  >
    <span>
      World
    </span>
  </FormItem>
  <FormItem
    key="1"
    label="Hello-world"
  >
    <span>
      <div>
        Hello World
      </div>
    </span>
  </FormItem>
  <FormDivider
    key="2"
  />
</Form>
`;
